docs/conf/portland/2022/badge-flair.rst

```diff
@@ -0,0 +1,6 @@
+:template: {{year}}/generic.html
+
+Badge Flair Contest
```

Comment:
Is there a way to hide this with a flag? We haven't decided whether we'll run it and it might create inequities for the virtual attendees